deployment/example-workload/crd.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: contour.heptio.com/v1alpha1
+kind: Route
+metadata: 
+  name: finance-route
+  namespace: default
+spec: 
+  host: heptio.com
+  routes: 
+    - pathPrefix: /
+      upstreams: 
+        - lbHealthCheck: 
+            intervalSeconds: 1
+            path: /status
+          serviceName: www
```

Comment:
Seems like this should point to the `kuard` service that gets deployed in this example?

Reply:
haha, was just updating to do that now =)